documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -16,7 +16,7 @@ You must upgrade to the next release without skipping a release, for example, fr
 
 . In the {ocp-short} web console, click *Operators* -> *Installed Operators* -> *{operator-name-ui}* -> *Subscription*.
 
-. Change the update channel to *release-v2.2.0*.
+. Change the update channel to the correct release.
```

Comment:
@istein1 -- I made this change since the text referred to version 2.2. It it OK? Should I add "for example, *release-v.2.3.0" ?  I am guessing that the UI will say  *release-v.2.3.0"  ... Please check that everything is OK for version 2.3.

Do we need version by version upgrade instructions? I don't think so, but I want to raise the issue.